docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -114,16 +199,31 @@ This numbering and the round-robin partition distribution assures the even repli
-| Web Modeler                    | Web Modeler is a standalone component that is not covered in this guide. Modelling applications can operate independently outside of the orchestration clusters.                                                                                                                                                                                                                                                                                                            |
+| **Aspect**            | **Details**                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| :-------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Installation methods  | <p><ul><li>For **Kubernetes** we recommend using a dual-region Kubernetes setup with the [Camunda Helm chart](/self-managed/installation-methods/helm/install.md) installed in two Kubernetes clusters.</li><li>For **other platforms**, using alternative installation methods (for example, with docker-compose) is not covered by our guides.</li></ul></p>                                                                                                              |
+| Traffic Management    | <p>**Hybrid Active-Active/Active-Passive Architecture**</p><p><ul><li>**Data Layer**: Active-active replication with zero RPO</li><li>**User Traffic**: Active-passive routing to prevent conflicts</li><li>**All Components**: Must be operational in both regions</li></ul></p>                                                                                                                                                                                           |
+| Identity Support      | <p>Identity, including multi-tenancy and Role-Based Access Control (RBAC), has specific requirements in dual-region setups:</p><p><ul><li>**External Identity Provider**: Recommended approach using external IdP</li><li>**Shared Database**: Single PostgreSQL instance accessible from both regions</li><li>**Not Supported**: Independent Identity instances per region</li></ul></p>                                                                                   |
```

Comment:
I'd leave it out atm since we're not actively supporting it and not sure what the future atm is with the new embedded Identity.
This just opens doors to more requests since the documentation says so.

I'd leave the same as before. `Identity, including multi-tenancy and Role-Based Access Control (RBAC), is currently unavailable in this setup.`

Consulting makes the impossible possible.